### PR TITLE
Add a send() method for directly accessing ga.send().

### PR DESCRIPTION
**Pull request by karanjthakkar**
Possible fix for #64 and #65

### CODE REVIEW

**Comment by karanjthakkar** [comment]
@yangshun Thoughts?

**Comment by karanjthakkar** [comment]
@yangshun Bump!

File: src/index.js [hunk 1]
```diff
@@ -94,6 +94,23 @@ var ReactGA = {
   },
 
   /**
+   * send:
+   * Clone of the low level `ga.send` method
+   * WARNING: No validaions will be applied to this
```
Comment (yangshun) [inline src/index.js]: Typo in validations

**Comment by yangshun** [comment]
This is great! Please fix the typo and it's good to be merged

**Comment by karanjthakkar** [comment]
@yangshun done!

**Pull request merged**